### ISSUE TITLE
Scroll blockers don't work

### ISSUE THREAD

**Issue by phobos2077** [alexeevdv/falltergeist#251]
Scroll blockers should:
- not be visible while in-game, only in map editor
- block scrolling by checking against hex in the center of the screen

**Comment by alexeevdv** [comment]
Yeah.... also we are planing to hide all area that is not visible in original game. For example, we will see black area around Arroyo temple on high resolutions. That also should be implemented using scroll blockers as visible area markers.

**Comment by vvbalashoff** [comment]
For hiding invisible areas there is a
https://github.com/falltergeist/falltergeist/issues/286
opened

**Comment by phobos2077** [comment]
I think we better do this sooner than later, because no scroll blocking is annoying. We can add temporary config variable to disable it.
